.github/workflows/test-integration-rosa.yaml

```diff
@@ -0,0 +1,359 @@
+      - '.github/workflows/test-integration-rosa.yaml'
+      - '.github/actions/workflow-vars/**'
+      - '.github/actions/chart-test-recipes/**'
+      - '.github/workflows/fixtures/clusters/rosa-hcp-on-demand/**'
+      - '.github/workflows/fixtures/modules/**'
+      - '.tool-versions'
+  workflow_call:
+    inputs:
+      cluster_name:
+        description: The unique identifier of used in the cluster name, will be random if not provided.
+        default: ""
+        type: string
+      identifier:
+        description: The unique identifier of used in the deployment hostname.
+        required: true
+        type: string
+      camunda-helm-git-ref:
+        default: main
+        type: string
+      caller-git-ref:
+        default: main
+        type: string
+      persistent:
+        description: |
+          Keep test deployment after the workflow is done.
+          NOTE: All persistent deployments will be deleted frequently to save costs!
+        default: false
+        type: boolean
+      flows:
+        description: "JSON formated list of flows"
+        default: '["install", "upgrade"]'
+        type: string
+      test-enabled:
+        default: true
+        type: boolean
+      extra-values:
+        description: Pass extra values to the Helm chart.
+        default: ""
+        type: string
+
+env:
+  TEST_PERSISTENT: ${{ inputs.persistent }}
+
+  # please keep those variables synced with test-integration-template.yml
+  TEST_AWS_REGION: "eu-central-1"
+  TF_S3_BUCKET: "${{ secrets.DISTRO_CI_OPENSHIFT_TFSTATE_BUCKET }}"
+
+  TEST_HOSTNAME_BASE: ci.distro.ultrawombat.com
+  # Docker Hub auth to avoid image pull rate limit.
+  TEST_CREATE_DOCKER_LOGIN_SECRET: "TRUE"
+  TEST_DOCKER_USERNAME: ${{ secrets.DISTRO_CI_DOCKER_USERNAME_DOCKERHUB }}
+  TEST_DOCKER_PASSWORD: ${{ secrets.DISTRO_CI_DOCKER_PASSWORD_DOCKERHUB }}
+  # Camunda registry auth to access WebModeler Docker image since it's not public.
+  TEST_DOCKER_USERNAME_CAMUNDA_CLOUD: ${{ secrets.DISTRO_CI_DOCKER_USERNAME_CAMUNDA }}
+  TEST_DOCKER_PASSWORD_CAMUNDA_CLOUD: ${{ secrets.DISTRO_CI_DOCKER_PASSWORD_CAMUNDA }}
+
+  INPUTS_IDENTIFIER: "${{ inputs.identifier }}"
+  INPUTS_PERSISTENT: "${{ inputs.persistent || contains(github.event.*.labels.*.name, 'test-persistent') }}"
+  INPUTS_TEST_ENABLED: "${{ inputs.test-enabled || 'true' }}"
+  INPUTS_CALLER_GIT_REF: "${{ inputs.caller-git-ref || github.event.pull_request.head.sha || github.sha }}"
+  INPUTS_CAMUNDA_HELM_GIT_REF: "${{ inputs.camunda-helm-git-ref || github.event.pull_request.head.sha || github.sha }}"
+  INPUTS_EXTRA_VALUES: "${{ inputs.extra-values || '' }}"
+  INPUTS_FLOWS: ${{ inputs.flows || '[\"install\", \"upgrade\"]' }}
+
+  ASSIGNEES_ISSUE_ERROR: "aabouzaid, jessesimpson36, hamza-m-masood, drodriguez-305"
+
+# limit to a single execution per actor of this workflow
+concurrency:
+  group: "${{ github.workflow }}-${{ github.actor }}"
+
+jobs:
+  define_matrix:
```

Comment:
For consistency, let's use the dash `-` instead of the underscore `_` in the keys as we have for the rest of the workflows.